org.eclim.jdt/java/org/eclim/plugin/jdt/command/dependency/AddDependencyCommand.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2005 - 2016  Eric Van Dewoestine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.eclim.plugin.jdt.command.dependency;
```

Comment:
Done - i moved all my classes from the dependency package including the test cases to the classpath package.
See changes in the next commit.